apisix/plugins/hmac-auth.lua

```diff
@@ -74,6 +74,11 @@ local consumer_schema = {
             type = "boolean",
             title = "whether to keep the http request header",
             default = false,
+        },
+        enable_encode = {
```

Comment:
Better to use `encode_uri_param` so people can know what is encoded.

Reply:
done.